models/Document/PageSnippet.php

```diff
@@ -110,7 +111,6 @@ abstract class PageSnippet extends Model\Document
 
     public function __construct()
     {
-        $this->elements = & $this->editables;
         $this->inheritedElements = & $this->inheritedEditables;
```

Comment:
This should be removed then, too, I think (and be made private)

Reply:
@jdreesen 👍